### ISSUE TITLE
Current directory when running via BSP is set to `.scala`

### ISSUE THREAD

**Issue by tgodzik** [VirtusLab/scala-cli#443]
To reproduce:

Run below snippet from within metals and from cli.

```scala
// using lib com.lihaoyi::os-lib:0.7.8

@main
def main =
  println(os.pwd) 
end main
```

**Comment by alexarchambault** [comment]
That must be because we run bloop from that directory, and use it to handle run main class requests (for now at least).

**Comment by tgodzik** [comment]
We can change the generated configuration:
```
"directory":"/home/tgodzik/Documents/advent-of-code/day2/.scala","workspaceDir":"/home/tgodzik/Documents/advent-of-code/day2/.scala"
```

**Comment by lwronski** [comment]
@tgodzik Thanks for reporting, good point, after changed configuration for bloop it will print current directory without `.scala`. It was fixed in `0.0.9`.
